api/src/test/java/com/epam/pipeline/manager/metadata/MetadataApiServiceTest.java

```diff
@@ -16,6 +16,7 @@
 
```

Comment:
This test class should be removed. But before we can do that we should either ensure that all tests in the class are covered by some tests in other classes or move them from the deleting class to some other classes.